modules/ROOT/pages/index.adoc

```diff
@@ -5,71 +5,19 @@ endif::[]
 :imagesdir: ../assets/images/
 :page-aliases: general:getting-started:anypoint-exchange.adoc, ex-ref-raml.adoc
 
-Anypoint Exchange provides the benefit of being able to discover, share, and incorporate assets and resources into your applications. Anypoint Exchange helps you create API developer portals, view and test APIs, simulate data to APIs (Mocking Service), create assets, and use API Notebooks to describe and test API functions.
+Anypoint Exchange is your curated catalog of APIs and other assets, including API groups, API specification fragments, policies, and more.
 
-== Get Started
+image::ex2-ui.png[Exchange interface]
 
-. From the public https://www.anypoint.mulesoft.com/exchange/[Exchange], click an asset and view its description.
-. Click Login to view a private Exchange for your organization.
-. After you log in, click Public Portal in your organization to view public APIs available to anyone.
+Use Exchange to discover, share, and incorporate assets and resources into your applications. Exchange helps you create API developer portals, view and test APIs, simulate sending data to APIs by using the mocking service, create assets, and use API Notebooks to describe and test API functions.
 
-After you log into Exchange, you can create assets for APIs, connectors, templates, examples, API Notebooks, and custom assets.
-
-== Try APIs
-
-. Click All Types > REST APIs and choose any API.
-. Click the asset and notice how the API functions appear in the left
-navigation area. Exchange lets you view the functions in an API.
-. Click a function and more information appears. At the right side of the screen, you can test the API by entering sample values to send to the API and get responses from the API.
-
-== Need More Information on an Asset?
-
-Click the Helpful Links item in the left navigation area for links to documentation.
-
-== Exchange Content
-
-Exchange assets are accessible in Anypoint Studio (connectors, templates, and examples), API Manager (APIs, API Groups, and policies), and Runtime Manager (APIs), and you can publish APIs from Design Center to Exchange.
-
-MuleSoft certifies its public content and reviews and approves partner connectors. All private content is internal to an organization and not visible outside that organization. Customers are responsible for any private content they publish within their Exchange instance.
-
-== Language Support
+Exchange assets are accessible in Anypoint Studio (connectors, templates, and examples), API Manager (APIs, API groups, and policies), and Runtime Manager (APIs), and you can publish APIs from Design Center to Exchange.
 
 Using multiple languages with Unicode is supported fully throughout Exchange and in all metadata fields, including asset names, descriptions, markdown pages, categories, tags, and reviews.
 
-[[content-types]]
```

Comment:
hi, is the removed content no longer applicable or located in another topic?

Reply:
This PR moves the "Content Types" list from the landing page to the new page "Asset Types". It never really made sense for it to be part of the landing page.